bitnami/nginx/README.md

```diff
@@ -90,6 +90,23 @@ The following tables lists the configurable parameters of the NGINX Open Source
 | `service.externalTrafficPolicy`            | Enable client source IP preservation                                                        | `Cluster`                                                    |
 | `service.loadBalancerIP`                   | LoadBalancer service IP address                                                             | `""`                                                         |
 | `service.annotations`                      | Service annotations                                                                         | `{}`                                                         |
+| `ldapDaemon.enabled`                       | Enable LDAP Auth Daemon proxy                                                               | `false`                                                      |
```

Comment:
Because unlike other Charts, NGINX requires a sidecar daemon that serves as proxy between the actual LDAP server and itself. I guess I can change it to `ldap`, but I did want to establish a difference between other [charts](https://github.com/bitnami/charts/blob/c7751eb5764e468e1854b58a1b8491d2b13e0a4a/bitnami/postgresql/templates/statefulset.yaml#L218) that simply set env vars to work and this one.

Reply:
Ok, now I understand, that makes sense! thanks!